GVRf/Framework/jni/engine/importer/assimp_importer.h

```diff
@@ -44,7 +66,8 @@ class AssimpImporter: public HybridObject {
     unsigned int getNumberOfMeshes() {
         return assimp_importer_->GetScene()->mNumMeshes;
     }
-
+    void sceneRecursion(aiNode* node, const aiScene* aScene, std::shared_ptr<Scene> scnPtr, JNIEnv * env, jobject obj, jobject bitmap, aiMatrix4x4 accumulatedTransformation);
```

Comment:
Actually sceneRecursion() is used as a local c++ function that can be freely stripped off any unnecessary parameters.